assets/js/builder/Views/SettingsFields.js

```diff
@@ -2,7 +2,7 @@
  * Model settings fields view
  *
  * @since 3.17.0
- * @version 4.7.0
+ * @version [version]
```

Comment:
```suggestion
 * @version 4.7.0
```
Nothing was changed here :D

Reply:
yup (changed but reverted) 👨‍💻